config/v1/types_infrastructure.go

```diff
@@ -20,6 +20,10 @@ type Infrastructure struct {
 }
 
 type InfrastructureSpec struct {
+	// CloudProvider is the IaaS provider that is running the cluster. The value
+	// should be a valid kubelet --cloud-provider value.
+	CloudProvider string `json:"cloudProvider,omitempty"`
```

Comment:
You think this should be mutable?  I expected it to be immutable.

Reply:
Yeah; is there more to do with docs or tags in that case?

Reply:
Relocated to status